src/ArrayObject/PHP81Implementation.php

```diff
@@ -0,0 +1,495 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib\ArrayObject;
+
+use ArrayAccess;
+use Countable;
+use IteratorAggregate;
+use Laminas\Stdlib\Exception\InvalidArgumentException;
+use Serializable;
+use Traversable;
+use UnexpectedValueException;
+
+use function array_keys;
+use function asort;
+use function class_exists;
+use function count;
+use function get_class;
+use function get_object_vars;
+use function gettype;
+use function in_array;
+use function is_array;
+use function is_callable;
+use function is_object;
+use function is_string;
+use function ksort;
+use function natcasesort;
+use function natsort;
+use function serialize;
+use function sprintf;
+use function strpos;
+use function uasort;
+use function uksort;
+use function unserialize;
+
+/**
+ * Custom framework ArrayObject implementation
+ *
+ * Extends version-specific "abstract" implementation.
+ */
+class PHP81Implementation implements IteratorAggregate, ArrayAccess, Serializable, Countable
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API

Reply:
when `class_alias` is used, psalm would assume the aliased class as internal as well as it cannot make a difference afaik. We might check that before marking it internal.

Reply:
Agreed, that is indeed a problem :thinking: